ocs_ci/templates/ocp-deployment/install-config-vsphere-upi.yaml.j2

```diff
@@ -3,6 +3,9 @@ baseDomain: {{ base_domain | default('qe.rh-ocs.com') }}
 metadata:
   name: '{{ cluster_name }}'
 networking:
+  clusterNetwork:
+    - cidr: {{ cluster_network_cidr | default('10.128.0.0/14') }}
+      hostPrefix: {{ cluster_host_prefix | default(23) }}
```

Comment:
do we have cluster_host_prefix defined in default_config.yaml ?

Reply:
I guess its not required, can use param cluster_host_prefix in another file which can override value 23

Reply:
The default value in default_config is not needed as it will be taken from | default(23) .  But if we have some config file, it will be taken from there instead of it.